Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -0,0 +1,176 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Shared
+
+protocol ZoomPageBarDelegate: AnyObject {
+    func zoomPageDidPressClose()
+}
+
+class ZoomPageBar: UIView {
+    struct UX {
+        static let leadingTrailingPadding: CGFloat = 26
+        static let topBottomPadding: CGFloat = 18
+        static let stepperWidth: CGFloat = 222
+        static let stepperHeight: CGFloat = 36
+        static let stepperLeadingTrailingMargin: CGFloat = 12
+        static let fontSize: CGFloat = 16
+    }
+
+    weak var delegate: ZoomPageBarDelegate?
+
+    var tab: Tab
+    var isIpad: Bool
+
+    private let stepperContainer: UIStackView = .build { view in
+        view.axis = .horizontal
+        view.alignment = .center
+        view.distribution = .fillProportionally
+        view.layer.cornerRadius = 8
+        view.layer.shadowRadius = 4
+        view.layer.shadowOffset = CGSize(width: 0, height: 3)
+        view.layer.shadowOpacity = 1
+        view.clipsToBounds = false
+    }
+
+    private let zoomOutButton: UIButton = .build { button in
+        button.setImage(UIImage.templateImageNamed(ImageIdentifiers.subtract), for: [])
+        button.accessibilityIdentifier = "ZoomPage.zoomOut"
+        button.contentEdgeInsets = UIEdgeInsets(top: 6,
```

Comment:
This inset of 6 can probably be in the UX struct up here. Will be easier to modify if needed, all in once place